pkg/manager/impl/validation/execution_validator.go

```diff
@@ -62,6 +62,11 @@ func ValidateExecutionRequest(ctx context.Context, request admin.ExecutionCreate
 	if err := validateLiteralMap(request.Inputs, shared.Inputs); err != nil {
 		return err
 	}
+	if request.Spec.GetNotifications() != nil {
+		if err := validateNotifications(request.Spec.GetNotifications().Notifications); err != nil {
+			return err
+		}
+	}
```

Comment:
We can have launchplan metadata also use NotificationList and that way we can move L65 if check and just pass the NotificationList from both the places.

Reply:
Sorry missed this yeah i meant we should probably have this consistent in protos too